src/games/strategy/engine/framework/ui/NewGameChooser.java

```diff
@@ -51,20 +51,22 @@ private NewGameChooser(final Frame owner) {
     setupListeners();
     setWidgetActivation();
     updateInfoPanel();
+    WaitDialog loadingScreen = new WaitDialog(owner, "Loading Maps...");
+    refreshGameList();
+//    SwingUtilities.invokeLater(() -> {loadingScreen.setVisible(false);loadingScreen.dispose();}); dispose() is throwing a "Exception: null" exception
```

Comment:
Do a search for the word "Substance", there is a comment somewhere in the code that mentions a config that is needed for it to work well. I give it only a 30% chance that is the actual problem. Not sure otherwise, not 100% sure what substance even gives us TBH.